doc/DEVELOPERS.rst

```diff
@@ -1,13 +1,13 @@
----------------------------
+===========================
```

Comment:
using `-------` for top level is intentional for this file. See https://github.com/rst2pdf/rst2pdf/commit/6d6490acde1ff05c864936d730d66121e27b3f6c#diff-faab44affa9a75879bed7063892c32f5:

> DEVELOPERS.rst is included in the manual and so it needs to use the same
title levels as manual.rst such that it embeds into the manual below the
chapter level.

i.e. we can't change this without breaking the TOC in the PDF of the manual.

Reply:
Dropped this commit